deploy/configs/modules.yaml

```diff
@@ -17,14 +17,6 @@ modules:
     all:
       autoload: 'all'
       load_only_generated: true
-      environment:
-        set:
-          '${PACKAGE}_ROOT': '${PREFIX}'
-    # Don't set any environment (like CMAKE_ROOT), it'll interfere with
-    # library finding. `::` overrides with the empty set.
-    cmake:
-      environment:
-        set:: {}
     gcc:
```

Comment:
(copying from Slack discussion):

In the cluster environment, some variable like `FOO_HOME` or `FOO_ROOT` has been quite convenient (a standard thing). For example, if I have to build something with specific HDF5 or CUDA then I load/swap modules but my cmake/autoconf configure command looks like:

```
./configure --cuda-lib-dir=$CUDA_HOME/lib --hdf5-dir=$HDF5_HOME
```

Same when we do caliper thing:

```
cmake .. -DCaliper_Dir=$CALIPER_HOME/share/cmake/
```

So any one variable pointing to the install prefix should be ok. We haven't seen this problem for couple of years but only with py-torch + cmake. May be we should just blacklist CMake only?

Reply:
(from Slack)

I think that https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html?highlight=cmake_prefix_path is the more appropriate way to go and neither `_ROOT` nor `_HOME` should be set unless explicitly required.